catalyst-gateway/bin/src/service/common/objects/cardano/cip36.rs

```diff
@@ -60,12 +62,11 @@ impl Example for Cip36Reporting {
 }
 
 /// CIP36 Registration Data as found on-chain.
-#[derive(Object, Default)]
+#[derive(Object)]
 #[oai(example = true)]
 pub(crate) struct Cip36Info {
```

Comment:
Is it possbile to change `payment_address` to `payment_key`

Reply:
I believe the comment is wrong, and payment address is literally the payment address.
It should be cip19 encoded if it is, not a hex value.
I am yet to fix this area up in the API's
Will be part of future PR's.